index.html

```diff
@@ -0,0 +1,353 @@
+Traditional single-screen laptop form factor is excluded from the foldable device definition and web specifications that refer to this form factor specifically refer to it as a <dfn>traditional laptop</dfn>. Wikipedia has a more nuanced definition of laptop types, but for the purpose of [related web specifications](#related-web-specifications) [all laptop types](https://en.wikipedia.org/wiki/Laptop#Types) are considered traditional laptops.
+</aside>
+
+### Dual-screen device
+
+A <dfn data-export="">dual-screen device</dfn> has two or more physical internal screens made of solid materials placed together to form a larger foldable virtual screen. A [=dual-screen device=] is an instance of a <dfn data-export="">multi-screen device</dfn> with two screens. A defining characteristic of the [=dual-screen device=] form factor is a physical hinge that can rotate from 0 degree to at most 360 degrees that forms a physical gap in between the screens. The range of the hinge movement as well as the width of the gap varies depending on the hardware design. The screens separated by the physical hinge are made of solid materials, typically glass, and as such are not flexible. This design requires web developers to work around the fact there is a physical gap between the screens on both sides of the fold.
+
+### Foldable-screen device
+
+A <dfn data-export="">foldable-screen device</dfn> has one or multiple screens made of foldable material that can bend so that the each screen can fold onto itself. The hinge can typically rotate from 0 degree to 180 degrees depending on the flexible screen materials’ capability of sustaining the bend radius. The material around the area of the screen that can bend is often softer than on the other parts of the screen. To differentiate from a dual-screen device, a foldable-screen device does not have a physical gap between the screens that form the logical regions. This makes it easier for existing web content to be presented without visual distractions when the foldable-screen device is folded open into so-called flat posture.
+
+### Multi-monitor setups
+
+A <dfn data-export="">multi-monitor setup</dfn> is a traditional mainstream setup in which a device (foldable or not) has one or more monitors connected to it (wired or not). Major desktop operating systems provide affordances to the user to arrange the logical position of the monitors as seen by the OS to match the physical location of the monitors as selected by the user. In this configuration, it is typical to mix and match monitors of different physical sizes in both landscape and horizontal configurations to optimize the screen estate for the usage model. Power users may also make use of advanced software tools to create virtual zones to which application windows can snap into (see e.g. [Microsoft PowerToys FancyZones](https://github.com/microsoft/PowerToys/wiki/FancyZones-Overview)). The virtual zones provide a similar abstraction to the [=window segments=].
```

Comment:
I attempted to rewrite this for clarity (and included a potential re-naming of the term), perhaps some of this can be broken out to describe a more generalized multi-display arrangement term?:

> An <dfn data-export="">extended-screen device</dfn> is any device with one or more physical displays connected to expand the default visual workspace. This includes traditional desktop, laptop, and mobile devices, with additional displays connected beyond the single screen typical of these form factors. Additional displays are often monitors, televisions, or projectors connected by a wire, but also includes other display form factors and connection technologies.

> Major desktop operating systems allow users to customize the logical arrangement of multiple screens, including their relative arrangement and other attributes. The arrangement may or may not reflect the placement and dimensions of underlying physical hardware. In this configuration, it is typical to mix and match monitors of different physical sizes and resolutions, in both landscape and portrait configurations, to optimize the screen space for the usage model. This free-form composition of screens is distinct from the nature of dual-screen devices, which have well-defined relationships between physical displays, and from folding-screens, which expands the operational modes of a single physical display.


Perhaps this content is more fitting as a note in in the window segments definition?

> Power users may also make use of advanced software tools to create virtual zones to which application windows can snap into (see e.g. [Microsoft PowerToys FancyZones](https://github.com/microsoft/PowerToys/wiki/FancyZones-Overview)). The virtual zones provide a similar abstraction to the [=window segments=].

Reply:
Replaced the section with this rewrite proposed by @michaelwasserman with editorial changes.